flink-connectors/flink-connector-pulsar/pom.xml

```diff
@@ -36,13 +36,15 @@ under the License.
 	<packaging>jar</packaging>
 
 	<properties>
-		<pulsar.version>2.8.0</pulsar.version>
+		<pulsar.version>2.9.1</pulsar.version>
```

Comment:
How is this version bump necessary?

Basically, we should not bump version in a patch version, especially which can introduce new features.

Reply:
We have to use this version bump because the underlying API is introduced on Pulsar 2.9.x.